wasmtime-jni/src/io_github_kawamuray_wasmtime_Store/imp.rs

```diff
@@ -55,4 +55,9 @@ impl<'a> JniStore<'a> for JniStoreImpl {
             JObject::null().into_inner()
         })
     }
+
+    fn native_g_c(env: &JNIEnv, this: JObject) -> Result<(), Self::Error> {
+        let mut store = interop::get_inner::<Store<StoreData>>(&env, this)?;
+        Ok(store.gc())
```

Comment:
This works, because `store.gc()` returns `()`, but to avoid misleading readers that this is returning something a meaningful value, let's return unit explicitly.

```rust
store.gc();
Ok(())
```